quick_task/resources/views/staff/index.blade.php

```diff
@@ -0,0 +1,47 @@
+@extends('staff.layout')
+
+@section('content')
+    <a href="{{ url('lang/vi') }}">Tiếng việt</a>
+    <a href="{{ url('lang/en') }}">Tiếng anh</a>
```

Comment:
```suggestion
    <a href="{{ url('lang/vi') }}">Tiếng việt</a>
    <a href="{{ url('lang/en') }}">English</a>
```
hoặc nếu bn muốn hiển thị theo ngôn ngữ người dùng thì phải dùng @lang ở đây